src/component/jhi-boolean.component.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ Copyright 2013-2017 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see https://jhipster.github.io/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+import {Component, Input} from '@angular/core';
+
+@Component({
+    selector: 'jhi-boolean',
+    template: `<i class="fa fa-lg"
+                  [ngClass]="{'fa-check': value,
```

Comment:
Make it possible to override the icon. Look at sort directive

Reply:
That way you can remove the nothingIfFalse option and just pass empty when particular icon is not needed